workflows/files/ZWECONF.xml

```diff
@@ -2104,8 +2128,13 @@ echo '        # this is required if storage mode is infinispan' >> "${instance-z
 echo '        jgroups:' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 #if (${instance-components_caching_service_storage_mode} == "infinispan" )
 echo '          port: ${instance-components_caching_service_storage_infinispan_jgroups_port}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '          host: ${instance-components_caching_service_storage_infinispan_jgroups_host}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '          keyExchange: ' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '            port: ${instance-components_caching_service_storage_infinispan_jgroups_keyExchange_port}' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
 #else
 echo '          port: 7600' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
+echo '          keyExchange: ' >> "${instance-zowe_runtimeDirectory}/zowe.yaml"
```

Comment:
There should be also the host in the else branch

Reply:
It can't be empty? My point is that if the user wouldn't specify it when they are running the workflow, but then they would like to, they can forget that this variable should to be set. What do you think?